profiler/src/ProfilerEngine/Datadog.Linux.ApiWrapper/functions_to_wrap.c

```diff
@@ -59,13 +60,210 @@ unsigned long long dd_inside_wrapped_functions()
     return functions_entered_counter;
 }
 
+#if defined(__aarch64__)
+const char* DdDotnetFolder = "linux-arm64";
+const char* DdDotnetMuslFolder = "linux-musl-arm64";
+#else
+const char* DdDotnetFolder = "linux-x64";
+const char* DdDotnetMuslFolder = "linux-musl-x64";
+#endif
+
+char* crashHandler = NULL;
+
+const char* getLibraryPath()
+{
+    Dl_info dl_info;
+
+    if (dladdr((void*)getLibraryPath, &dl_info))
+    {
+        return dl_info.dli_fname;
+    }
+    
+    return NULL;
+}
+
+int isAlpine()
+{
+    if (access("/etc/alpine-release", F_OK) == 0)
+    {
+        return 1;
+    }
+
+    return 0;
+}
+
+char* appendToPath(const char* folder, const char* suffix)
```

Comment:
How do people live like this, in this day and age?

Reply:
I know 😭